main.go

```diff
@@ -28,7 +29,27 @@ var (
 	}
 )
 
+// We supply some context from API Gateway to decide which signing key to use.
```

Comment:
Ditto for this comment, `golint` will trigger on this one - mind having it start with `APIGatewayContext contains some ...`